src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -239,6 +240,7 @@ else if (snDiff < 2 || snDiff > (-3 & 0xFFFF))
                 }
                 else if (snDiff < 3 || snDiff > (-4 & 0xFFFF))
                 {
+                    // FIXME or this.
```

Comment:
so do we think that the frequent log messages were only coming from the other condition (the one that you fixed after boris' comment)? or will this one still print a lot too?